index.html

```diff
@@ -1725,6 +1726,69 @@ <h3>Validity Period</h3>
 }
         </pre>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
```

Comment:
There are [many lists of date/time falsehoods](https://github.com/kdeldycke/awesome-falsehood#dates-and-time) that all programmers should familiarize themselves with.  And date and time handling in general is a large topic.  Listing only a few random examples of time handling issues here seems out of place.  Can this text be more general rather than listing examples?  If anything, I'd say the best place for non-trivial examples would be a test suite.

Reply:
> Can this text be more general rather than listing examples?

It could, but I'd be guessing at what sort of "more general" text you're looking for. If you can suggest something concrete, I might be able to incorporate it.

Reply:
I'm not sure where to take this.  It seems unnecessary to remind implementers to process dates and times correctly.  Beyond representation requirements, why is any processing advice needed?

Reply:
I'll raise another PR to try and remove the text and see if it gets traction. At this point, people are grumbling to the opening paragraph, but are not objecting. I'm going to merge it as is, and then try to remove it and see if people are supportive. I'll tag both of you on that PR.